src/components/exercise8/joke.jsx

```diff
@@ -0,0 +1,8 @@
+export default function Joke(props) {
```

Comment:
In my view, it would be beneficial to destructure these props in order to enhance code readability and comprehensibility.

Reply:
Thank you for pointing that out. and i have put them into considerations